gymnasium/gymstatic
# Create a featured courses "feed"

## What this PR does:

- adds a "page" that will provide a (json) description of the currently featured courses
- generates based on course IDs specified in front matter (`featured_courses: GYM-016, GYM-104, GYM-015`)

Closes gymnasium/tracker#80

### Testing:
1. Visit: https://deploy-preview-464--thegymcms.netlify.app/feeds/featured-courses.json
2. Copy the result
3. Paste into https://jsonlint.com/
3. Verify that the JSON is valid
